style.css

```diff
@@ -0,0 +1,204 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+:root {
+    view-transition-name: root;
+}
+
+html {
+    display: block;
+}
+
+.header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 9px 400px;
+    width: 1920px;
+    height: 70px;
+    color: #ffffff;
+    margin: 0 auto;
+    gap: 10px;
+    border-bottom: 1px;
+}
+
+.logo{
+    width: 153px;
+    height: 51px;
+    margin: 0 auto;
+    display: block;
+    cursor: pointer;
+}
+
+.login{
+    width: 128px;
+    height: 48px;
+    border-radius: 8px;
+
+    gap: 10px;
+    margin: 0 200;
+    display: block;
+}
+
+.headline-wrap{
+    display: flex;
+    justify-content: center;
+    align-items:center;
+    padding-top: 200px;
+    width: 1920px;
+    height: 540px;
+    margin: 0 auto;
+    background-color: #CFE5FF;
+    width: 100%;
+}
+
+.headline{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 1110px;
+    height: 340px;
+    gap: 7px;
+    margin: 0 auto;
+}
+
+.headline-content{
+    display: flex;
+    position: relative;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 357px;
+    height: 260px;
+    padding: auto 60px;
+    gap: 32px;
+}
+
+.headline-heading{
+
+    margin: 0 auto;
+    height: 112px;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 56px;
+    font-family: Pretendard;
+    color: #374151;
+}
+
+.headline-button{
+    text-decoration: none;
+    width: 357px;
+    height: 56px;
+    border-radius: 40px;
+    padding: 16px 120px;
+    background-color: #3B82F6;
+    color: #F9FAFB;
+    font-size: 20px;
+    font-weight: 500;
+    text-align: center;
+    font-family: Pretendard;
+    cursor: pointer;
+}
+
+.headline-content a {
+    text-decoration: none;
+    }
+
+.headline-image{
+    display: flex;
+}
+
+.wrap{
+    display: flex;
+    justify-content: center;
+    width: 1920px;
+    height: 2160px;
+    position: relative;
+    top: 610px;
+    padding: 138px 466px;
+    gap: 10px;
+    margin: 0 auto;
+}
+
+.content{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 988px;
+    height: 444px;
+    border-radius: 12px;
+    gap: 138px;
+    margin: 0 auto;
+}   
```

Comment:
❗️ 수정요청
해당 .wrap 요소가 html 흐름을 벗어나 띄워져 있고 특정 height를 가지고 있기 때문에
footer의 privacyPolicy, FAQ, SNS 링크들을 가려 클릭할 수 없게 합니다.

또한 특정 요소들을 position 속성을 이용해 배치하는 것은 css 작성을 어렵게하므로 추천드리지 않습니다.

<img width="1664" alt="스크린샷 2025-02-27 오후 4 21 23" src="https://github.com/user-attachments/assets/0663d382-7e34-4f28-9c6b-0120a36d3e82" />

따라서 내부 요소에 의해 자연스럽게 width, height값을 가지게 되므로 아래의 값만 남겨두시는 것을 추천드려요.

```suggestion
.wrap{
    display: flex;
    justify-content: center;
    gap: 10px;
}

// content 요소도 height 값이 너무 작게 입력되어 있어 디자인대로 나오지 않아 height, width값을 제거해주었습니다.
.content {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    border-radius: 12px;
    gap: 138px;
}   
```